src/biogeophys/TemperatureType.F90

```diff
@@ -1163,6 +1167,12 @@ subroutine InitAccBuffer (this, bounds)
     call init_accum_field (name='T10', units='K', &
          desc='10-day running mean of 2-m temperature', accum_type='runmean', accum_period=-10, &
          subgrid_type='pft', numlev=1,init_value=SHR_CONST_TKFRZ+20._r8)
+    call init_accum_field (name='SOIL10', units='K', &
+         desc='10-day running mean of 3rd layer soil temp.', accum_type='runmean', accum_period=-10, &
+         subgrid_type='pft', numlev=1,init_value=SHR_CONST_TKFRZ)
```

Comment:
I wonder about the generality of making this depend on the 3rd soil layer, since we sometimes use very different soil layer structures. Should there be some logic in initialization that chooses the layer closest to some desired depth, and make this variable depend on soil temperature of that chosen layer?